webviz_subsurface/_models/inplace_volumes_model.py

```diff
@@ -168,33 +219,110 @@ def set_initial_property_columns(self) -> None:
                 pvt = "BO" if vol_column == "STOIIP" else "BG"
                 self._property_columns.append(pvt)
 
-        self._dataframe = self.compute_property_columns(self._dataframe)
+    def _add_sensitivity_columns(self) -> None:
+        """Add sensitivity information columns from the parameters to the
+        dataframe, and raise error if not all ensembles have sensitivity data"""
+
+        self._parameterdf["SENSTYPE"] = self._parameterdf.apply(
+            lambda row: find_sens_type(row.SENSCASE)
+            if not pd.isnull(row.SENSCASE)
+            else np.nan,
+            axis=1,
+        )
+        sens_params_table = self._parameterdf[self.SENS_COLUMNS]
+
+        # TODO do this per ensemble?
```

Comment:
Do we have an issue on this? If not create one perhaps in order to remember :slightly_smiling_face: